### DISCUSSION TITLE
Masking groups?

### DISCUSSION THREAD

**Discussion by ListenerJubatus** [vcync/modV#884] [Q&A]
I'm trying to use modV to create a UV-mapped visualizer for a VJ performance in a VRChat world. I'm trying to make a group that renders _only_ in the stripes on the side like a mask. I did make a PNG texture that was transparent except for where the stripes are to try and use source/destination, but I'm not sure if it worked?

![Screen Diagram](https://github.com/vcync/modV/assets/6346433/cfee3746-62ef-40bd-b5ee-5e7f4ef87842)

**Comment by HorstBaerbel** [comment]
Late to the party, but this should work using the appropriate blend mode between groups
